app/adapters/alma_adapter/marc_response.rb

```diff
@@ -6,28 +6,52 @@ class MarcResponse
     # @param bibs [Alma::BibSet]
     def initialize(bibs:)
       @bibs = bibs
-      remove_suppressed!
```

Comment:
I like the refactor but it does appear to change the value of `bibs` which is a public method. Is it not used anywhere else in the codebase?

Reply:
@hackartisan It is not, because `bibs` is an `Alma::BibSet` and everywhere we've worked on this so far we've wanted MARC records.